web-client/src/app/stores/session/session.service.ts

```diff
@@ -0,0 +1,16 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { tap } from 'rxjs/operators';
+import { SessionStore } from './session.store';
+
+@Injectable({ providedIn: 'root' })
+export class SessionService {
+
+  constructor(private sessionStore: SessionStore, private http: HttpClient) {
+  }
+
+  get() {
+    return this.http.get('').pipe(tap(entities => this.sessionStore.update(entities)));
```

Comment:
I am not sure I understand need for an http call here. Is this how it would look if we were using a conventional API?

Reply:
not needed, removing it.

Reply:
Ah ok, thanks!